onnxruntime/core/mlas/lib/platform.cpp

```diff
@@ -34,25 +34,20 @@ MLAS_PLATFORM MlasPlatform;
 #define _XCR_XFEATURE_ENABLED_MASK 0
 #endif
 
-inline
-uint64_t
+inline uint64_t
 MlasReadExtendedControlRegister(
-    unsigned int ext_ctrl_reg
-    )
-{
+    unsigned int ext_ctrl_reg) {
 #if defined(_WIN32)
-    return _xgetbv(ext_ctrl_reg);
```

Comment:
This function ended up with the broken 2 space formatting style. Can you restore platform.cpp as it is in master? From what I can tell, you don't have any changes in this file.

Reply:
Resolved.  Copied platform.cpp from master.